src/simpla-element-behavior.html

```diff
@@ -10,185 +10,218 @@
 
   window.SimplaBehaviors = window.SimplaBehaviors || {};
 
-  SimplaBehaviors.Element = {
-    properties: {
+  SimplaBehaviors.Element = function(config) {
+    let {
+          type,
+          getCallback,
+          setCallback,
+          dataProperties,
+          updateOn
```

Comment:
And then there would be no need for events, because there's no reason to update Simpla if the data property didn't change. The only need for events is if the 'properties' you're giving this behavior aren't properties, as is the case with simpla-text. In that case you're misusing the behavior, IMO. It expects the things it's getting to be properties, not a getter funciton.